### PR TITLE
Disbursement journal

### PR DESCRIPTION
**Pull request by ibrechin**

### CODE REVIEW

**Comment by ibrechin** [comment]
retest this please

File: mtp_bank_admin/apps/bank_admin/tests/__init__.py [hunk 1]
```diff
@@ -94,11 +172,51 @@ def __call__(self, actual):
...
+
+
+@contextmanager
+def temp_file(data):
+    with tempfile.TemporaryFile() as f:
+        f.write(data)
+        yield f
+
+
+def assert_called_with(testcase, url, method, expected_data):
```
Comment (ushkarev) [inline mtp_bank_admin/apps/bank_admin/tests/__init__.py]: looks a lot like a base object method with `self` -> `testcase`

File: mtp_bank_admin/apps/bank_admin/disbursements_config.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        'column': 'W',
+        'value': '0210'
+    },
+    'cost_centre': {
+        'column': 'X',
+        'value': '{prison_ledger_code}'
+    },
+    'account': {
+        'column': 'Y',
+        'value': '2617902085'
```
Comment (ushkarev) [inline mtp_bank_admin/apps/bank_admin/disbursements_config.py]: maybe should be a setting?

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Pull request closed without merging**